### PR TITLE
Add warning banner to FSx documentation

### PR DESCRIPTION
**Pull request by balaramesh**

### CODE REVIEW

**Review by juliantap — APPROVED** [review]
@balaramesh this looks good to me. Is this intended to merge now (after gnarl review)?

**Comment by balaramesh** [comment]
Thanks @juliantap. Once Chuck takes a look, we can merge this.

File: trident-use/trident-fsx.adoc [hunk 1]
```diff
@@ -104,6 +104,8 @@ NOTE: Do not specify `dataLIF` for the `ontap-san` and `ontap-san-economy` drive
 
 WARNING: When using Amazon FSx for NetApp ONTAP with Astra Trident, the `limitAggregateUsage` parameter will not work with the `vsadmin` and `fsxadmin` user accounts. The configuration operation will fail if you specify this parameter.
 
+WARNING: FSx for NetApp ONTAP volumes come preconfigured with a SnapMirror. Deleting PVCs will not succeed, as Trident cannot modify or break them. Users will need to delete their PVs manually and clean them up on FSx.
```
Comment (juliantap) [inline trident-use/trident-fsx.adoc]: @balaramesh technically the official product name is Amazon FSx for ONTAP. You probably don't need "NetApp" before ONTAP.

**Comment by juliantap** [comment]
@gnarl @ @balaramesh are we good to merge this change?

**Comment by balaramesh** [comment]
I say we merge it

**Comment by juliantap** [comment]
Moved this to https://github.com/NetAppDocs/trident-internal/pull/46

**Comment by juliantap** [comment]
Closing this PR. Replaced with https://github.com/NetAppDocs/trident/pull/88

**Pull request closed without merging**